tests/tensor_search/test_utils.py

```diff
@@ -234,9 +234,12 @@ def test_parse_lexical_query(self):
             ('', ([], [])),
             ('"cookie"', (["cookie"], [])),
             ('"朋友"', (["朋友"], [])),
-            ('"', ([], ['"'])),
-            ('"""hello', ([], ['"""hello'])),
-            ('""" python docstring appeared"""', ([], ['"""', 'python', 'docstring', 'appeared"""'])),
+
+            # Badly formatted double quotes.
+            # Treat every 2 as a pair, bad/unpaired quotes become whitespace.
+            ('"', ([], [])),
+            ('"""hello', ([], ['hello'])),
+            ('""" python docstring appeared"""', ([], ['python', 'docstring', 'appeared'])),
             ('""', ([''], [])),
```

Comment:
what will be returned if the search term is empty string?

Reply:
This is already covered here in this test case: `('', ([], [])),`. No optional or required terms.